File: BarbeariaBackend/barber-helper-backend/src/main/java/br/com/projeto/barberhelper/service/ServicoService.java
package br.com.projeto.barberhelper.service;

import java.util.List;

import br.com.projeto.barberhelper.generic.Service;
import br.com.projeto.barberhelper.model.Servico;

public interface ServicoService extends Service<Long, Servico> {
    List<Servico> listarServicosDTO();
}

File: BarbeariaFrontend/FrontendWeb/barber-helper-angular/src/app/view/servico/servico.service.ts
import { Injectable } from '@angular/core';
import {HttpClient} from "@angular/common/http";
import { BaseService } from 'src/app/arquitetura/service/base.service';
import { Servico } from 'src/app/arquitetura/modelo/servico.model';

@Injectable({
  providedIn: 'root'
})
export class ServicoService extends BaseService<Servico>{

  constructor(httpClient: HttpClient) {
    super( httpClient, 'servico');
  }

  listarServicosDTO() {
    return this.httpClient.get<any>(`${this.API}/${this.path}/listar-dtos`);
  }
}
